content/en/integrations/guide/source-code-integration.md

```diff
@@ -581,6 +581,30 @@ If you're using the GitHub integration, click **Connect to preview** on error fr
 
 [101]: https://app.datadoghq.com/security/appsec
 
+{{% /tab %}}
+{{% tab "Dynamic Instrumentation" %}}
+
+You can see full source code files in **Dynamic Instrumentation** when creating or editing an instrumentation (dynamic log, metric, span, or span tags).
```

Comment:
Nit: I know that you link to it below as well, but I would take the opportunity to also add a link to the DI docs from this line, as so:

```suggestion
You can see full source code files in [**Dynamic Instrumentation**][102] when creating or editing an instrumentation (dynamic log, metric, span, or span tags).
```

WDYT?